scripts/install-linux.sh

```diff
@@ -362,6 +371,40 @@ ensure_opkssh_user_and_group() {
     fi
 }
 
+# get_te_download_path
+# Checks the INSTALL_VERSION to determin where to download the TE file to download
+#
+# Outputs:
+#   The URL to download the TE file to use
+get_te_download_path() {
+    local te_url version
+    if [[ "$INSTALL_VERSION" == "latest" ]]; then
+        version=$(wget --server-response --max-redirect=0 -O /dev/null "https://github.com/${GITHUB_REPO}/releases/latest" 2>&1 | sed -n -E 's/^  Location: .*\/tag\/v([0-9.]+).*/\1/p')
+    else
+        # Remove the 'v' prefix if present
+        version="${INSTALL_VERSION#v}"
```

Comment:
why remove the v? Is this to ensure that users can supply `v0.8.0` or `0.8.0`?

Reply:
I remove the v because it's not needed when comparing semver versioning, clean major, minor and patch numbering